package.json

```diff
@@ -28,12 +28,15 @@
   },
   "homepage": "https://github.com/MasterLemon2016/LeanMoodBackend#readme",
   "devDependencies": {
+    "@types/bluebird": "^3.5.8",
```

Comment:
I see in `./src/dbs/index.ts` we are using ES6 promises. Should we uninstall `bluebird and typings`?